netbox/extras/forms/bulk_import.py

```diff
@@ -85,6 +90,17 @@ class Meta:
             'name', 'description', 'extra_choices', 'order_alphabetically',
         )
 
+    def clean_extra_choices(self):
+        if isinstance(self.cleaned_data['extra_choices'], list):
+            data = []
+            for line in self.cleaned_data['extra_choices']:
+                try:
+                    value, label = re.split(r'(?<!\\):', line, maxsplit=1)
+                except ValueError:
+                    value, label = line, line
+                data.append((value, label))
```

Comment:
We also need to normalize any escaped colons (`\:`) when saving the data.
```suggestion
                value = value.replace('\:', ':')
                label = label.replace('\:', ':')
                data.append((value, label))
```

(This should also be applied to the model form for #14467.)

Reply:
It needed to be saved with the escaped character as doing the save/edit cycle would replace it.  This has a change to replace the label when it is displayed in the model form using the custom field.

Reply:
The escape character should never be saved to the database; it's relevant only for processing raw input. I don't follow what you mean about the save/edit cycle, but we'll have to figure out a different approach to ensure the normalized values are stored.

Reply:
The save/edit cycle meant if you edit and save a choice-set then re-edit it.  You either have to save the escape characters or else re-add them when edit.  I've changed to re-add them when you edit so they aren't saved in the database.